components/core/Profile.js

```diff
@@ -103,78 +101,495 @@ const STYLES_DESCRIPTION = css`
+        ? this.state.publicSlates
+        : followingSlates?.length
+        ? followingSlates
+        : null;
+    let username = this.state.slateTab === 0 ? data.username : null;
+
+    let following = subscriptions
+      .filter((relation) => {
+        return !!relation.target_user_id;
+      })
+      .map((relation) => {
+        let button = (
+          <div css={STYLES_ITEM_BOX} onClick={(e) => this._handleClick(e, relation.id)}>
+            <SVG.MoreHorizontal height="24px" />
+            {this.state.contextMenu === relation.id ? (
+              <Boundary
+                captureResize={true}
+                captureScroll={false}
+                enabled
+                onOutsideRectEvent={(e) => this._handleClick(e, relation.id)}
+              >
+                {relation.target_user_id === this.props.viewer?.id ? (
+                  <PopoverNavigation
+                    style={{
+                      top: "40px",
+                      right: "0px",
+                    }}
+                    navigation={[
+                      {
+                        text: "Copy profile URL",
+                        onClick: (e) =>
+                          this._handleCopy(e, `https://slate.host/${relation.user.username}`),
+                      },
+                    ]}
+                  />
+                ) : (
+                  <PopoverNavigation
+                    style={{
+                      top: "40px",
+                      right: "0px",
+                    }}
+                    navigation={[
+                      {
+                        text: "Copy profile URL",
+                        onClick: (e) =>
+                          this._handleCopy(e, `https://slate.host/${relation.user.username}`),
+                      },
+                      {
+                        text: this.props.viewer?.subscriptions.filter((subscription) => {
+                          return subscription.target_user_id === relation.target_user_id;
+                        }).length
+                          ? "Unfollow"
+                          : "Follow",
+                        onClick: this.props.viewer
+                          ? (e) => this._handleFollow(e, relation.target_user_id)
+                          : () => this.setState({ visible: true }),
+                      },
+                    ]}
+                  />
+                )}
+              </Boundary>
+            ) : null}
+          </div>
+        );
+        return (
+          <UserEntry
+            key={relation.id}
+            user={relation.user}
+            button={button}
+            onClick={() => {
+              this.props.onAction({
+                type: "NAVIGATE",
+                value: this.props.sceneId,
+                scene: "PROFILE",
+                data: relation.user,
+              });
+            }}
+            external={this.props.external}
+            url={`https://slate.host/${relation.user.username}`}
```

Comment:
I'll make this change, but in the future, it's better to do links as url={`/${relation.user.username}`} so that they work on dev server as well. Otherwise when you test out these links on dev, it'll redirect you to slate.host rather than localhost

Also simpler in case the domain ever changes in the future

Reply:
I see ok